test/unit/behaviours/skip-step.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const reqres = require('reqres');
```

Comment:
I'd change this to use `hof-util-reqres` just because that's liable to be more useful in other unit tests.